server.js

```diff
@@ -5,11 +5,11 @@ const app = express();
 const PORT = 3000;
 const distDir = path.join(__dirname, 'dist');
 
-app.get('/', function(req, res) {
-  res.sendFile(path.join(distDir, 'auth.html'));
-});
 app.use(express.static(`${__dirname}/dist/`));
 
-app.listen(PORT, function () {
-  console.log(`Example app listening on port ${PORT}!`);
-});
\ No newline at end of file
+app.use('/', (req, res) => {
+  res.sendFile(path.join(distDir, 'index.html'));
+});
+app.listen(process.env.PORT || PORT, () => {
+  console.log(`Example app listening on port ${process.env.PORT || PORT}!`);
```

Comment:
Лучше вынести в константу
```suggestion
const PORT = process.env.PORT || 3000;
app.listen(PORT, () => {
  console.log(`Example app listening on port ${PORT}!`);
```